<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>980</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-color:#D0DAE5;
}

QTextBrower{
background-color:#D0DAE5;
}

QLineEdit{
background-color:#D0DAE5;
}

QWidget{
background-color:#D0DAE5;
}

QStackedWidget{
background-color:#D0DAE5;
}

QHeaderView{
background-color:#D0DAE5;
}

QLabel{
font-size:12px;
font-family:Microsoft YaHei;
background-color:#D0DAE5;
}

QPushButton{         /*按钮自适应拉伸背景*/
border-width: 2px 15px 2px 15px;
background-color:#2884D8;
color: #FFFFFF;
font-size:12px;
font-family:Microsoft YaHei;
}

QGroupBox{
font-size:12px;
font-family:Microsoft YaHei;
border:1px solid rgb(0, 0, 0); 
}</string>
  </property>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>0</width>
     <height>0</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>0</width>
     <height>0</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_test_results">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>961</width>
     <height>521</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QTableWidget" name="tableWidget_test_results"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>146</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_start">
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="iconQrc.qrc">
        <normaloff>:/newPrefix/icon/start1.ico</normaloff>:/newPrefix/icon/start1.ico</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>31</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_restart">
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="iconQrc.qrc">
        <normaloff>:/newPrefix/icon/restart1.ico</normaloff>:/newPrefix/icon/restart1.ico</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>31</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButton_close">
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="iconQrc.qrc">
        <normaloff>:/newPrefix/icon/close1.ico</normaloff>:/newPrefix/icon/close1.ico</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>31</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>961</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,2">
    <item>
     <widget class="QGroupBox" name="groupBox_test_modules">
      <property name="title">
       <string>测试项目</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="verticalSpacing">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,3">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_test_duration">
              <property name="font">
               <font>
                <family>Microsoft YaHei</family>
                <pointsize>-1</pointsize>
                <kerning>false</kerning>
               </font>
              </property>
              <property name="text">
               <string>00:00:00</string>
              </property>
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QTreeWidget" name="treeWidget">
           <property name="headerHidden">
            <bool>true</bool>
           </property>
           <column>
            <property name="text">
             <string>1</string>
            </property>
           </column>
           <item>
            <property name="text">
             <string>协议控制和转换模块</string>
            </property>
            <property name="flags">
             <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_test_resource">
      <property name="title">
       <string>测试资源</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <property name="topMargin">
        <number>20</number>
       </property>
       <item row="0" column="0">
        <widget class="QTableWidget" name="tableWidget_test_resource">
         <property name="gridStyle">
          <enum>Qt::DotLine</enum>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>120</number>
         </attribute>
         <row>
          <property name="text">
           <string>1</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>2</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string/>
          </property>
         </column>
         <column>
          <property name="text">
           <string>名称</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>类型</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>编号/型号</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>数量</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item row="0" column="1">
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item row="0" column="2">
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item row="0" column="3">
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item row="0" column="4">
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item row="1" column="1">
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item row="1" column="2">
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item row="1" column="3">
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item row="1" column="4">
          <property name="text">
           <string>2</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="iconQrc.qrc"/>
 </resources>
 <connections/>
</ui>
